notebooks/tel_and_site/subsys_req_ver/m2/SITCOM2102_M2Hexapod_fault_study.ipynb

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
<p>Instead of writing this up here, consider adding a link to the official documentation:</p><p><a href="https://docs.python.org/3/library/logging.html#logging-levels" target="_blank">https://docs.python.org/3/library/logging.html#logging-levels</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/190/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>